apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -611,6 +613,18 @@ suite('navigation >', function() {
         }).then(done, done);
       });
     });
+
+    suite('setReady', function() {
```

Comment:
This trick is only for non-split view, so only add it in the non-split test suite.